spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -90,12 +90,32 @@
     end
   end
 
-  describe "#unschedule" do
+  describe "#schedule" do
+    let(:step_by_step_page) { create(:draft_step_by_step_page) }
+
     before :each do
-      stub_user.permissions << "Scheduling"
+      stub_scheduling_permissions
+      stub_publishing_api_for_scheduling
+    end
 
-      allow(Services.publishing_api).to receive(:get_content)
-        .and_return(content_item(step_by_step_page))
+    it "sets `scheduled_at` to a datetime" do
+      schedule_publishing(step_by_step_page, "2030-04-20 10:26:51 London")
```

Comment:
Nitpick: When the date and time inputs are split, users won't be able to add "seconds", so it might be easier not to assign them here, rather than having to remove them from the test later.